Extension/package.nls.json

```diff
@@ -66,6 +66,12 @@
             "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
         ]
     },
+    "c_cpp.configuration.maxSymbolSearchResults.markdownDescription": {
+        "message": "The maximum number of results to show for GoTo Symbol. The default is 5000.",
```

Comment:
The command is called "Go to Symbol", so let's use that capitalization.

It doesn't look like we use markdown for this setting, so maybe a normal description is ok, but I think it's ok to leave it if you want.